### ISSUE TITLE
Can't disable screenshot binding for f12

### ISSUE THREAD

**Issue by soredake** [hrydgard/ppsspp#17559]
### Game or games this happens in

All

### What area of the game / PPSSPP

![image](https://github.com/hrydgard/ppsspp/assets/5204968/9fab46ff-37f3-40f3-8d85-3b5672b58d5f)

There is no bind for screenshot in controls, yet pressing f12 will result in screenshot take.

### What should happen

Screenshot should not be taken if no bind is selected for f12 function.

### Platform

Windows

### Mobile phone model or graphics card

Asus RX 560

### PPSSPP version affected

v1.15.4

### Graphics backend (3D API)

Direct3D 11

### Checklist

- [X] Test in the [latest git build](https://buildbot.orphis.net/ppsspp/) in case it's already fixed.
- [X] [Search for other reports](https://github.com/hrydgard/ppsspp/search?q=my+issue&type=issues) of the same issue.
- [X] Try resetting settings or older versions and include if the issue is related.
- [X] Try without any cheats and without loading any save states.
- [X] Include logs or screenshots of issue.

**Comment by hrydgard** [comment]
Yeah, this is caused by us having both fixed bindings for the Windows menu stuff, and the configurable bindings that are in-game only. It gets confusing.

Though, I'm not sure why it's bad that F12 takes a screenshot?

**Comment by unknownbrackets** [comment]
There is already an option to disable the standard Windows shortcuts which will disable this and other UI things that are common on Windows.

-[Unknown]

**Comment by soredake** [comment]
> Though, I'm not sure why it's bad that F12 takes a screenshot?

Cause i use https://github.com/SteamGridDB/steam-rom-manager to add stuff to steam and play with steam overlay which by default does screenshot with F12 too, resulting in two screenshots taken, PPSPPP and steam.

**Comment by soredake** [comment]
> There is already an option to disable the standard Windows shortcuts which will disable this and other UI things that are common on Windows.

![image](https://github.com/hrydgard/ppsspp/assets/5204968/ee94ab85-5321-4895-ac93-1d48baf398ee)

This option does not have a word about "Windows" or "Screenshot" so i wouldn't guess that i need to disable this in order to disable screenshot function. Anyway, it's confusing to have action for bindings that are not set.

**Comment by LunaMoo** [comment]
It should not have a word "windows" in it, PPSSPP is multi platform and this disables all standard keyboard shortcuts, don't expect every hardcoded key to have separate off setting.😅

 PPSSPP historically always had standard mapping when no mapping was present for more common stuff to avoid situation where the users being dumb, lock themselves from being able to do anything by removing controls.

**Comment by hrydgard** [comment]
It's hard to name things :) If you have a better idea please post it.

**Comment by soredake** [comment]
Better for me would be not to have such option and just have default bindings in controls, but that is only my opinion.

**Comment by hrydgard** [comment]
That's fair enough, and maybe we should extend the binder to run not only in-game.

**Comment by soredake** [comment]
Started using ppsspp again, still, this is very un-intuitive to have such option (without clear description of what shortcuts it have) instead of just having preconfigured shortcuts for this actions.

**Comment by NABN00B** [comment]
Question, is there still a use-case for these hardcoded bindings now? Is the F8 pause different from the rebindable pause functionality?

**Comment by hrydgard** [comment]
Hardcoded bindings fit the Windows model with shortcuts being displayed in the menu. It's a little tricky to make those dynamic, although of course technically possible.

Basically this is just history, and it's tricky to make changes that won't break things for somebody out there...

**Comment by NABN00B** [comment]
Strictly speaking from user perspective here.

It makes sense for key combinations like
- Alt + Return
- Alt + F4
- Ctrl + `n` (wherever these come from) -- for some reason these aren't always displayed in the menu bar?

but for the single F-keys, I don't know...
- F1 used to change rendering resolution, but it's gone now 🎉 
- F7 for frame skipping -- doesn't seem practically useful and doesn't even work for me
- F2-F4 for savestate stuff -- these can be remapped, and would make more sense to reuse these F-keys akin to #19878
- F8 for pause -- this can now be remapped, which is why I asked in my previous post if there's a difference between the two
- F11 for fullscreen -- this can now be remapped, and should rather be set as a default keybind rather than being hardcoded
- F12 for screenshot -- same as F11

... maybe these made more sense when the emulator used to be less customizable.

If anything, the pause menu should be hardcoded to Esc. Or better yet auto-mapped to Esc if it isn't bound to any key under controls.

At least we do have the option of disabling them, which is very nice. 😄

**Comment by hrydgard** [comment]
Yeah, maybe it's time for a cleanup of those...

**Comment by Panderner** [comment]
Pressing F12 also overlaps toggle debugger button by default.

**Comment by hrydgard** [comment]
I've decided to just remove the built-in F12 shortcut for screenshots, as it's a bit intrusive indeed, and you can just rebind it if you want.
